services/codeclimate.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+---
+
+# [Code Climate](https://codeclimate.com)
+
+Code Climate is a service that incorporates fully-configurable test coverage and maintainability data throughout the development workflow, making quality improvement explicit, continuous, and ubiquitous.
+
+We use Code Climate for quality metrics and code coverage.
+
+## Quality Metrics
+
+To setup quality metrics (static analysis), the following steps are executed.
+
+  - We add the `.codeclimate.yml` file to the repository. This file defines  the tools used for the analysis.
+  - In Code Climate, we add the project.
+	- In Code Climate, the [GitHub Pull Requests](https://docs.codeclimate.com/v1.0/docs/github) integration is configured.
+
+## Code Coverage *experimental*
+
+TO BE ADDED
```

Comment:
Is this correctly still pending?

Reply:
Yes. I will fill it out once we will have the integration in place.